starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,459 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 >= number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+}
+
```

Comment:
This correctly done!
Another way to do it which will be cleaner is by  
returning Math.max(a,b) which returns the largest number